### DISCUSSION TITLE
can't remove device

### DISCUSSION THREAD

**Discussion by DocMAX** [nefarius/DsHidMini#165] [Q&A] [answered]
is it possible that you driver depends on the same bluetooth dongle it has been installed on? because if i use a differend one, ds4 is not detected anymore with ds4windows. also i cannot remove the device (remove failed). how can i fix this?

**Comment by nefarius** [comment]
Thanks for ignoring the issue templates. Really love the extra moderation work.

I assume you're talking about BthPS3, you remove it by uninstalling it, like any other app. And yes, ofc. if you switch Bluetooth dongles, you need to pair the controller again, by plugging it in via USB at least once after you swap your radio, that's a limitation of how the DS3 "pairs".

Cheers

**Comment by DocMAX** [comment, marked as answer]
sorry, i think the template wasn't relevant in my case. thanks for answering anyway. i needed to remove the old dongle in hidden devices in device manager. now ds4 controller works again.

**Comment by DocMAX** [comment]
with ps3 controllers pairing works even if i change to another bt dongle. but this doesnt work with ps4 controllers. i have to remove the device from device manager and then pair with the other dongle. maybe we can workaround this with software?

**Reply by nefarius** [reply]
No, that is the only way to do it.
